packages/core/factories/src/useProductFactory.ts

```diff
@@ -38,10 +39,11 @@ export function useProductFactory<PRODUCT, PRODUCT_SEARCH_PARAMS>(
     };
 
     return {
-      products: computed(() => products.value),
-      totalProducts: computed(() => totalProducts.value),
       search,
-      loading: computed(() => loading.value)
+      productGetters: factoryParams.productGetters,
```

Comment:
makeComputedGetters should be used here

Reply:
then the factory would need a type directly from integration, wouldn't it be better to send ready getters?

Reply:
the idea of factory is to take care of repeatable logic and not force integrations to do it on their own :)